packages/react/src/primitives/types/message.ts

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
```

Comment:
@dbanksdesign  and I were just talking about this; our prop table generation relies on the types living in `/src/primitivers/types`. Additionally, both this primitive and Breadcrumbs don't currently show types correctly in our prop tables, so either way that script needs to be updated.

I can move the types now but will need to disable the prop table tab on this page. Maybe thats ok since it's not showing them all anyways so it's not all that useful? It's basically just showing the [base component props](https://messageprimitive-message.dvmvffzts1tcu.amplifyapp.com/react/components/message?tab=props) that all primitives have, same for Breadcrumbs on live site right now. We definitely need to fix the table generation though either way for these new composable primitives.

Reply:
I realized the prop table generation doesn't rely on `/src/primitives/types` but rather on `/src/primitives/components.ts` and it basically looks at the types of all the exported members from that file and sees if they extend from the Primitive type. So as long as you properly type your primitive and export it from `components.ts` the props-table should work.